node/parachain/src/cli.rs

```diff
@@ -98,7 +98,7 @@ pub struct RunCmd {
 
 	/// Public identity for participating in staking and receiving rewards
 	#[structopt(long, parse(try_from_str = parse_h160))]
-	pub account_id: Option<H160>,
+	pub author_id: Option<H160>,
```

Comment:
TODO change `--account-id` to `--author-id` in `tests`, `scripts` and elsewhere

Reply:
Thanks for doing that